src/olympia/abuse/models.py

```diff
@@ -345,14 +346,16 @@ def process_decision(
         ).without_parents_if_their_children_are_present()
         self.policies.add(*policies)
         action_helper = self.get_action_helper(existing_decision, override=override)
-        action_occured = action_helper.process_action()
+        log_entry_or_true = action_helper.process_action()
         if (
             existing_decision != decision_action
             or decision_action == self.DECISION_ACTIONS.AMO_APPROVE
         ):
             action_helper.notify_reporters()
-        if action_occured:
-            action_helper.notify_owners()
+        if log_entry_or_true:
+            action_helper.notify_owners(
+                log_entry_id=getattr(log_entry_or_true, 'id', None)
+            )
```

Comment:
Can you think of a better way of doing this ? Returning either a boolean, or an activity log, or None is quite confusing (setting a property out of nowhere isn't that much better...)

Reply:
mmm, well I had `log_entry_id` be a property on the `CinderAction` instance before but setting it was hacky with `resolve_job`.  I could still do that and have something like `log_entry_id = log_entry_id or self.log_entry_id` in `notify_owners`?

Reply:
I'm thinking eventually that all the `process_action`s will return an activity log instance, but that'll need some rewriting where the functionality is already delegated somewhere else.  Really, it's only useful to know the log id for add-on targets anyway - it's used for the activitylog emails, and that's only a thing where there is an add-on involved.

Could have a tuple, yep.